README

```diff
@@ -116,6 +116,23 @@ To install to system run `make install`. By default `make install` will install
     dependencies: *pkg-config*(build-time), *libnl-3* and *libnl-genl-3*
     default: *check*
 
+## runtime dependencies:
+`htop` has a set of fixed runtime dependencies, which is kept as minimal as possible:
+* `ncurses`, libraries for terminal handling (wide character support)
+* `libcap`, *Optional* user-space interfaces to the POSIX 1003.1e
+* `libdl`, if not building static and support for some of the optional libraries is enabled
+* `libsensors`, readout of temperatures and CPU speeds.
+* `systemd`
+
+`htop` checks for the availability of the actual runtime lib as `htop` runs.
+
+**BSD**    
+On most *BSD systems you also have `kvm` as a static requirement to read all the kernel information.
+
+
+For more info [configure.ac](configure.ac)
```

Comment:
```suggestion
More information on required and optional dependencies can be found in [configure.ac](configure.ac).
```

Reply:
So all of the runtime dependencies are optional?

> ## Runtime optional libraries:
> `htop` has a set of fixed runtime optional libraries for more features, which is kept as minimal as possible:
> 
> * `ncurses`, libraries for terminal handling (wide character support)
> * `libcap`, user-space interfaces to the POSIX 1003.1e
> * `libdl`, if not building static and support for some of the optional libraries is enabled
> * `libsensors`, readout of temperatures and CPU speeds.
> * `systemd`

Reply:
* `ncurses` is always required.
* `libcap` is always required when `--enable-capabilities` was used to configure `htop`.
* `libsensors` is optional even when `--enable-sensors` was used to configure `htop`.
* `systemd` is optional when `--enable-static` was not used to configure `htop` (Linux only). If building statically and `libsystemd` is not found by `configure` support for the SystemD meter is disabled entirely.
* `libdl` is always required when support for to optionally load dependencies (i.e. `libsensors`, `systemd`) is present.